packages/manager/src/features/ObjectStorage/BucketDetail/BucketProperties.tsx

```diff
@@ -0,0 +1,113 @@
+import { Typography } from 'src/components/Typography';
+import { useFlags } from 'src/hooks/useFlags';
+import { useAccount } from 'src/queries/account/account';
+import { useObjectStorageBuckets } from 'src/queries/object-storage/queries';
+import { isFeatureEnabledV2 } from 'src/utilities/accountCapabilities';
+import { getQueryParamFromQueryString } from 'src/utilities/queryParams';
+
+import { BucketRateLimitTable } from '../BucketLanding/BucketRateLimitTable';
+import { BucketBreadcrumb } from './BucketBreadcrumb';
+import {
+  StyledActionsPanel,
+  StyledHelperText,
+  StyledRootContainer,
+  StyledText,
+} from './BucketProperties.styles';
+
+import type { ObjectStorageEndpointTypes } from '@linode/api-v4';
+
+interface Props {
+  bucketName: string;
+  clusterId: string;
+  endpointType?: ObjectStorageEndpointTypes;
+}
+
+export interface UpdateBucketRateLimitPayload {
+  rateLimit: string;
+}
+
+export const BucketProperties = React.memo((props: Props) => {
+  const { bucketName, clusterId, endpointType } = props;
+
+  const form = useForm<UpdateBucketRateLimitPayload>({
+    defaultValues: {
+      rateLimit: '1',
+    },
+  });
+
+  const {
+    formState: { errors, isDirty, isSubmitting },
+    handleSubmit,
+  } = form;
+
+  const location = useLocation();
+  const history = useHistory();
+  const prefix = getQueryParamFromQueryString(location.search, 'prefix');
+  const flags = useFlags();
+  const { data: account } = useAccount();
+
+  const isObjMultiClusterEnabled = isFeatureEnabledV2(
+    'Object Storage Access Key Regions',
+    Boolean(flags.objMultiCluster),
+    account?.capabilities ?? []
+  );
+
+  const { data: buckets } = useObjectStorageBuckets();
+
+  const bucket = buckets?.buckets.find((bucket) => {
+    if (isObjMultiClusterEnabled) {
+      return bucket.label === bucketName && bucket.region === clusterId;
+    }
+    return bucket.label === bucketName && bucket.cluster === clusterId;
+  });
+
+  const onSubmit = () => {
+    // TODO: OBJGen2 - Handle Bucket Rate Limit update logic once the endpoint for updating is available.
+    // The 'data' argument is expected -> data: UpdateBucketRateLimitPayload
+  };
+
+  return (
+    <FormProvider {...form}>
+      <BucketBreadcrumb
+        bucketName={bucketName}
+        history={history}
+        prefix={prefix}
+      />
+      <StyledText>{bucket?.hostname || 'Loading...'}</StyledText>
+
+      <StyledRootContainer>
+        <Typography variant="h2">Bucket Rate Limits</Typography>
+
+        {errors.root?.message ? (
+          <Notice text={errors.root?.message} variant="error" />
+        ) : null}
+
+        {/* TODO: OBJGen2 - We need to handle link in upcoming PR */}
+        <StyledHelperText>
+          Specifies the maximum Requests Per Second (RPS) for an Endpoint. To
+          increase it to High, open a <Link to="#">support ticket</Link>.
+          Understand <Link to="#">bucket rate limits</Link>.
```

Comment:
```suggestion
          Specifies the maximum Requests Per Second (RPS) for an Endpoint. To
          increase it to High, <SupportLink text="open a support ticket" />.
          Understand <Link to="#">bucket rate limits</Link>.
```

We're going to want to use the `SupportLink` component for this here. It will link us directly to the support ticket dialog. 

Additionally, we *could* pre-populate the ticket `title` and potentially, once M3-7946 is done, include the `entity` we're passing in. See [`CreateImageTab.tsx`](https://github.com/linode/manager/blob/develop/packages/manager/src/features/Images/ImagesCreate/CreateImageTab.tsx#L195-L203) for an example.

We typically also link the whole phrase "open a support ticket", so I've included that change in my suggestion here.

Reply:
I have added a `title` to the SupportLink for now.